lib/wallaroo/core/data_receiver/data_receiver.pony

```diff
@@ -183,17 +183,25 @@ actor DataReceiver is Producer
       @printf[I32]("Rcvd pipeline msg at DataReceiver\n".cstring())
     end
     if seq_id > _last_id_seen then
-      ifdef "resilience" and debug then
-        if (seq_id - _last_id_seen) != 1 then
-          @l(Log.crit(), Log.no_cat(), "seq_id %lu _last_id_seen %lu".cstring(), seq_id, _last_id_seen)
-        end
-        Invariant((seq_id - _last_id_seen) == 1)
+      if (seq_id - _last_id_seen) == 1 then
```

Comment:
If we're not in resilience mode, then we should enter this branch even if the gap is greater than 1. This is why the Invariant was inside that ifdef before.

Reply:
True, I've not been good about always checking `ifdef "resilience"` constraints. Fixed (I hope) by a45ebda66